tests/unit/test_sampler/fixtures/parent_span_context.py

```diff
@@ -5,8 +5,8 @@
 @pytest.fixture(name="parent_span_context_invalid")
 def fixture_parent_span_context_invalid():
     return SpanContext(
-        trace_id=00000000000000000000000000000000,
-        span_id=0000000000000000,
+        trace_id=0o0000000000000000000000000000000,
+        span_id=0o000000000000000,
```

Comment:
Very minor nit but could this be changed to hex since that's now the usual notation for these IDs